app/screens/home/maximize_screen.tsx

```diff
@@ -140,7 +141,14 @@ export default function MaximizeScreen({
             colorType="transparent"
             testID="challenge-image"
           >
-            <Image source={{ uri: postImage }} style={styles.image} />
+            <Image
+              source={
+                postImage
+                  ? { uri: postImage }
+                  : require("@/assets/images/challenge1.png")
+              }
```

Comment:
This is no use since a missing image already displays the message "No image to display"

Reply:
That is correct. I will fix it accordingly